examples/offloader.js

```diff
@@ -12,17 +14,17 @@ var fs = require('fs');
 
 var client = new SteamUser();
 var manager = new TradeOfferManager({
-	"steam": client, // Polling every 30 seconds is fine since we get notifications from Steam
```

Comment:
I personally prefer to enforce double quotes around keys in objects, so that it's also valid JSON. Also prevents any surprises from characters that can't be in (unquoted) keys.